Dockerfile.dev

```diff
@@ -0,0 +1,35 @@
+FROM node:13.10.1-slim
+
+RUN apt-get update
+RUN apt-get install -y yarn
+RUN apt-get install -y git
+RUN apt-get install -y libacl1-dev
+RUN apt-get install -y libncurses-dev
+
+#esy install
+RUN apt-get install -y curl
+
+#esy bootstrap
+RUN apt-get install -y clang
+RUN apt-get install -y make
+RUN apt-get install -y m4
+RUN apt-get install -y nasm
+RUN apt-get install -y libfontconfig1-dev
+RUN apt-get install -y libx11-dev
+RUN apt-get install -y libsdl2-dev
+RUN apt-get install -y libbz2-dev
+RUN apt-get install -y libgtk-3-dev
+
+RUN apt-get install -y opam
+RUN opam init
+RUN opam install dune
+
+RUN yarn global add esy@0.6.2
+
+WORKDIR /code
+
+RUN esy install
+RUN esy bootstrap
+RUN esy build
+
+CMD esy run
```

Comment:
On the `CMD` instruction, it is preferred to use arguments JSON notation.
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#cmd
https://github.com/hadolint/hadolint/wiki/DL3025
```dockerfile
CMD ["esy", "run"]
```